zerver/openapi/zulip.yaml

```diff
@@ -17079,6 +17164,64 @@ components:
         - to
         - topic
         - content
+    ScheduledMessage:
+      type: object
+      description: |
+        A dictionary for representing a scheduled message.
+      properties:
+        message_id:
+          type: integer
+          description: |
+            The unique ID of the scheduled message. It can be used to modify and
+            delete the scheduled message.
+        type:
+          type: string
+          description: |
+            The type of the scheduled message. Either unaddressed (empty string), "stream",
+            or "private" (for PMs and private group messages).
+          enum:
+            - ""
+            - stream
+            - private
+        to:
+          type: array
+          description: |
+            An array of the tentative target audience IDs. For "stream"
+            messages, this should contain exactly 1 ID, the ID of the
+            target stream. For private messages, this should be an array
+            of target user IDs. This cannot be an empty array since
+            scheduled messages are always addressed.
```

Comment:
This last sentence seems at variance with the `type` description, which says the scheduled message can be "unaddressed". I'm not sure how to reconcile those.

Reply:
Yeah, the "unaddressed" case isn't possible and should be removed from the documentation -- that's a situation that exists for drafts that looks to have been copy/pasted incorrectly.